### PR TITLE
add enrollments and submissions queries to CD2

### PR DESCRIPTION
**Pull request by jennlove-um**
These markdown files add a place to keep queries related to submissions and enrollments.

### CODE REVIEW

File: CD2/enrollments.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#Enrollments Queries
```
Comment (zqian) [inline CD2/enrollments.md:1]: improvements on Markdown syntax needed. Here is the [Markdown cheatsheet](https://www.markdownguide.org/cheat-sheet/)

insert a space in-between the markdown symbols and the text. For example, use "# Enrollments" vs "#Enrollments".

You can do preview using VisualStudio or https://markdownlivepreview.com/

File: CD2/enrollments.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+##Get enrollment dates for students in a specified course
+
+*This query finds the dates enrollments were created and updated as well as enrollment status for students in the course.*
+
+SELECT us.value.name as student_name
```
Comment (zqian) [inline CD2/enrollments.md:7]: I would recommend using the code block symbol:

```
SQL_CODE
```

File: CD2/enrollments.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    ,co.value.sis_source_id as SIS_course_ID
+    ,en.value.workflow_state as enrollment_state
+    ,en.value.created_at as enrollment_create_date
+    ,en.value.updated_at as enrollment_update_date
+    ,en.value.completed_at as enrollment_completion_cate
+FROM udp-umich-prod.canvas.enrollments as en
+JOIN udp-umich-prod.canvas.pseudonyms ps on en.value.user_id = ps.value.user_id
+JOIN udp-umich-prod.canvas.courses co on en.value.course_id = co.key.id
+JOIN udp-umich-prod.canvas.users us on en.value.user_id = us.key.id
+WHERE en.value.course_id = COURSE ID
```
Comment (zqian) [inline CD2/enrollments.md:21]: use <COURSE_ID> instead, to mark it as a parameter

File: CD2/submissions.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    ,su.value.published_grade
+FROM udp-umich-prod.canvas.assignments as ad
+    ,udp-umich-prod.canvas.submissions as su
+    ,udp-umich-prod.canvas.pseudonyms as ps
+    ,udp-umich-prod.canvas.users as us
+WHERE su.value.assignment_id = ad.key.id
+    AND su.value.user_id = ps.value.user_id
+    AND su.value.user_id = us.key.id
+    AND ad.value.context_id = COURSE ID
+    AND su.value.graded_at > TIMESTAMP('YYYY-MM-DDT00:00:00 UTC')
```
Comment (zqian) [inline CD2/submissions.md:22]: use <'YYYY-MM-DDT00:00:00 UTC'> instead, to mark this as a parameter

File: CD2/submissions.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    AND ad.key.id in (LIST OF ASSIGNMENT IDS)
```
Comment (zqian) [inline CD2/submissions.md:23]: use <[list of assignment IDs]> to indicate an array of params

File: CD2/submissions.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+#Assignment Submissions Queries
```
Comment (zqian) [inline CD2/submissions.md:1]: same improvements on Markdown format.

**Review by jennlove-um — COMMENTED** [review]
Fixes markdown syntax

**Review by jennlove-um — COMMENTED** [review]
Fixes markdown syntax

**Pull request merged**